dev-python/virtualenvwrapper/files/virtualenvwrapper-4.8.4-adjust-test-error-message.patch

```diff
@@ -0,0 +1,22 @@
+From a948d6f66f5705447a98afe3bea89988ceceaf5d Mon Sep 17 00:00:00 2001
```

Comment:
For all patches, please put a link to the source at the top (as well as any other relevant links, like a bug)

Reply:
I went with the commit url + pull request url. The only bug related to one of the commits was very lackluster but its also linked now.